### PR TITLE
New Adapter: AAX

### PR DESCRIPTION
**Pull request by product-aax**
Adds AAX Bidder Adapter.
Vendor ID: 720
Maintainer Email: [product@aax.media](mailto:product@aax.media)

### CODE REVIEW

**Comment by muuki88** [comment]
Documentation: https://github.com/prebid/prebid.github.io/pull/3696

**Comment by product-aax** [comment]
@muuki88 Can we assign someone to review this merge request ? Thanks !!

**Comment by muuki88** [comment]
Hi @product-aax 
Thanks for reaching out. I pinged @bretg as he manages the team. I'm merely an external contributor ☺️

**Review by And1sS — COMMENTED** [review]
Can be an alias for generic bidder: https://github.com/prebid/prebid-server-java/blob/78b12ff6fd5c62ec70b36d4da2aacfd91891e00b/src/main/java/org/prebid/server/bidder/GenericBidder.java

File: src/main/resources/bidder-config/aax.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+adapters:
+  aax:
+    endpoint: https://prebid.aaxads.com/rtb/pb/aax-prebid?src={{PREBID_SERVER_ENDPOINT}}
+    meta-info:
+      maintainer-email: product@aax.media
+      app-media-types:
+        - banner
+        - video
```
Comment (And1sS) [inline src/main/resources/bidder-config/aax.yaml:8]: Here https://github.com/prebid/prebid.github.io/pull/3696/files video is not mentioned. Is it docs mistake or is it configuration mistake(or maybe i missed something)?

**Comment by SerhiiNahornyi** [comment]
@product-aax small reminder on @And1sS question

File: src/main/resources/bidder-config/aax.yaml (hunk 1 above)
Comment (product-aax) [reply src/main/resources/bidder-config/aax.yaml:8]: We have added video in prebid doc

**Comment by product-aax** [comment]
@SerhiiNahornyi Sorry for the delay.

**Review by And1sS — APPROVED** [review]

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**